readme.md

```diff
@@ -0,0 +1,9 @@
+## Requirements
+* [Docker](https://www.docker.com/get-started)
+* JAVA
+
+## How to start
```

Comment:
ok, wszystko fajnie ale .... co to robi? jak powinienem wystartować aplikację? co się stanie po jej odpaleniu? jaki jest jej cel?
opisz to wszystko tutaj co chcesz zaprezentować, zrób jakąś funkcjonalność goddamnit! :D